packages/bpk-component-graphic-promotion/README.md

```diff
@@ -8,25 +8,25 @@ npm install bpk-component-graphic-promotion --save-dev
 
 ## Usage
 
-```js
+```tsx
 import React from 'react';
 import BpkGraphicPromo from 'bpk-component-graphic-promotion';
 
 export default () => (
   <BpkGraphicPromo
+    buttonText="Learn more"
```

Comment:
A bunch of these are just sorting alphabetically. It makes the diff a bit hard to read, but makes the existing code much easier to skim.